main.nf

```diff
@@ -9,21 +9,22 @@
 ----------------------------------------------------------------------------------------
 */
 
-
 def helpMessage() {
     log.info nfcoreHeader()
     log.info"""
+
     Usage:
 
     The typical command for running the pipeline is as follows:
 
     nextflow run nf-core/mhcquant --mzmls '*.mzML' --fasta '*.fasta' --vcf '*.vcf' --class_1_alleles 'alleles.tsv' --include_proteins_from_vcf --predict_class_1 --refine_fdr_on_predicted_subset -profile standard,docker
 
     Mandatory arguments:
-      --mzmls                           Path to input data (must be surrounded with quotes)
+      --mzmls                           Path to input data (must be surrounded with quotes)	
```

Comment:
Time to annotate all other parameters. This will be done in another PR somewhen I guess.